Gruntfile.js

```diff
@@ -334,9 +334,13 @@ module.exports = function(grunt) {
 			// Grab the relevant bits from the file contents
 			var fileContent = grunt.file.read( 'svg-min/' + svgFile );
 
+			// Grab the filename
+			var name = svgFile.split( '.' );
+			name = name[0];
 
 			// Add transparent rectangle to each file
 			fileContent = fileContent.slice( 0, fileContent.indexOf('viewBox="0 0 24 24">') + 20 ) +
+						'<title>' + name + '</title>' +
```

Comment:
So for example if the name is `gridicons-add-image`, with the above should return `Add Image`